apps/web/src/App/App.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { mockImplicitAccount } from "@umami/core";
+import { addTestAccount, makeStore } from "@umami/state";
+
+import { App } from "./App";
+import { render, screen } from "../testUtils";
+
+jest.mock("@chakra-ui/react", () => ({
+  ...jest.requireActual("@chakra-ui/react"),
+  useBreakpointValue: jest.fn(map => map["lg"]),
```

Comment:
It may not require this mock by wrapping in `UmamiProvider` inside the `customRender` function